app/views/wpcc/your_details/new.html.erb

```diff
@@ -118,24 +124,31 @@
               </div>
             </div>
           </div>
-          <div data-dough-popup-container class="popup-tip__container details__helper">
-            <p data-dough-popup-content class="popup-tip__content"><%= t('wpcc.details.salary.tooltip_html') %></p>
-            <%= render 'wpcc/tooltips/close' %>
-          </div>
+          <%= popup_tip_content options: {
+                                  text: t('wpcc.details.salary.tooltip_html'),
+                                  classname: 'details__helper',
+                                  tooltip_hide: t('wpcc.tooltip_hide')
+          } %>
```

Comment:
There's a lot of duplication on the popup_tip_content options hash, I'm wondering if it can be dried up with a helper.